WordPress/Classes/ViewRelated/Notifications/Views/NoteBlockHeaderTableViewCell.swift

```diff
@@ -32,14 +45,14 @@ class NoteBlockHeaderTableViewCell: NoteBlockTableViewCell {
 
 
     // MARK: - Public Methods
-    func downloadGravatarWithURL(_ url: URL?) {
-        if url == gravatarURL {
+    @objc func downloadGravatar(with url: URL?) {
```

Comment:
🤦‍♂️ I forgot to mention this on the previous review, but the `with` is not very Swifty IMO. I'd do `downloadGravatar(url:)` instead

Reply:
According to the API Design Guidelines:

```
Strive for Fluent Usage
Prefer method and function names that make use sites form grammatical English phrases.
```

And the example there says:

```
AudioUnit.instantiate(
  with: description, 
  options: [.inProcess], completionHandler: stopProgressBar)
```

This one is actually swifty 😄 

[Ref. here!](https://swift.org/documentation/api-design-guidelines/)

Reply:
😮 that's pretty clear, I don't know why I had interiorized the opposite. It still feels odd to me, but I'll take it.
However, I'd rename the Obj-C export `@objc(downloadGravatarWithURL:)`, it seems Swift isn't doing the conversion right here.